src/pages/components/content-group-simple.mdx

```diff
@@ -23,9 +24,24 @@ Content group &mdash; simple is ideal for grouping content within sections with
 
 ![Default](../../images/pattern/content-group-simple/default.jpg)
 
+## Content guidance
+
+| Field | Sub-component field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ---------- | ---------- |
+| Headline | | Short copy | Yes | 1 | XS (65 / 85)|
+| Paragraph | | Long copy | No | 1 | XXL (350 / 450)|
+| [Image with caption](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/image-with-caption) or [Video player](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/video-player) | | Component | No | 1 | - |
+| Content item | | Component | Yes | 2-6 | - |
+| | Headline | Short copy | Yes | 1 | XS (65 / 85) |
+| | Paragraph | Long copy | Yes | 1 | XXXXL (1600 / 2400) |
+| | [CTA (text)](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/cta) | Component | No | 1 | - |
```

Comment:
I kind of like the addition of sub component fields here. I think image or video is allowed at the content item level too if I'm not mistaken.

Reply:
@photodow Would we want them to do that with this component?

Reply:
@photodow Need your thoughts on this one.